Examples/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-alert": 0
```

Comment:
@vjeux Yes, I figured we could allow it for every file in examples instead of adding the rule per file since it was already used in a bunch of different files.